scania-digital-design-system/sdds
# Improvement - Grid update 12 col + more

**Describe pull-request**  
_Describe what the pull-request is about_

Update for the grid
- Gutter removed
- Padding value changed
- New breakpoints 
- All columns in alla breakpoints are 12
- Refactoring

Everything can be found in figma "new grid system"

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: [AB#827](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/827)

**How to test**  
_Add description how to test if possible_
1. Go to netlify / local
2. Check grid in storybook
3. Try all examples in all browsers
4. Try all breakpoints
5. Compare to figma

**Additional context**  
_Add any other context about the pull-request here._
